coral/src/app/accessibility.module.css

```diff
@@ -117,6 +117,10 @@ a:not(.Aquarium-Button):focus-visible {
   color: #4a4b57 !important;
 }
 
+:root [class*="typography-caption"]:not([class*="text-error-50"]) {
```

Comment:
Ah thank you! I suspect there are even more cases where this will cause issues :( We should really try to take these a11y changes upstream to `aquarium`

Reply:
There's still no plan an accessibility, unfortunately, so the chances of getting changes for colors from there are pretty low at the moment. Still should try! But with this fix, we should be ok, since it's bound to the `label` element. 

If we want to be 100% safe, we could add an element with a global class around that we use as entry point, but that's creating more DOM elements just for styling and is often not possible (e.g. when the style is on a nested element). But we could investigate it, at least than scoping is completely in our hands.

Reply:
Combined all the rules applying this color in one [8a8f9a4](https://github.com/Aiven-Open/klaw/pull/2006/commits/8a8f9a4e92be459f741a1a57ae37d47ce400982f)

Reply:
And actually tying it to the label role won't work, because the label in `SwitchGroup` is a `span` :(
<img width="952" alt="Screenshot 2023-11-22 at 11 31 50" src="https://github.com/Aiven-Open/klaw/assets/20607294/f63581c9-9acb-45e0-a7dd-4a1f4d07637d">

Reply:
UGH. 

We've the same rule for `legend` elements, so we could extend that too to also check children. Which should be pretty save, since there should not be children in a legend/label  that are not text elements. 

```
:root legend span:not([class*="error"]),
:root
  [class*="typography-caption"]:not([class*="text-error-50"]):not(
    [role="tooltip"]
  ),
 ```
 
is too brittle, since it targets all `typography-caption` classes with now two exceptions, but we would have to add more exceptions every time an element is wrongfully target, e.g. here

<img width="152" alt="Screenshot 2023-11-22 at 11 30 25" src="https://github.com/Aiven-Open/klaw/assets/943800/6f0aa32c-5bff-4b4e-9e61-95d36e43b890">

`label` / `legend` is very specific and we know that their children should be text, so that's safer.

Reply:
I mean, I think the core problem is that the color of `typograph-caption` is too light in every context, right? Small light greay text... So we may want to always override it.

Reply:
What is wild is that apparently `aquarium` DOES apply `grey-70` to the caption for input field descriptions... But not natively to `Typography.Caption`

<img width="1383" alt="Screenshot 2023-11-22 at 11 43 41" src="https://github.com/Aiven-Open/klaw/assets/20607294/51aff735-0974-41b5-922a-676621fb0aa3">

Reply:
I think this is sufficient and I think there are no conflicts with existing styles... [af43a29](https://github.com/Aiven-Open/klaw/pull/2006/commits/af43a2937f3cc1c56792fb86369772e4781181f5)

Reply:
I'm ok with that since it works, but for me to understand - where do you see the benefit of that to adding the style to children of label/legend (except error)? I feel that would scope it more clearly and also avoid that we need to add more exceptions to the style when we introduce a new caption element that should have a different colour. 

(Going ahead and approve)

Reply:
I'm not sure what you are asking ^^ You mean something more general than just `span` children of `legend` and `label`?

Reply:
Forget what I wrote 🤦 

I either opened the wrong link or github was not up to date. Did check out the branch :D 


I don't know why this works:

```
:root legend span:not([class*="error"]),
:root label span:not([class*="error"]) {
  /*represents grey-70 */
  color: #4a4b57 !important;
}
```

Instead of this (like it was before):

```

:root label span:not([class*="error"]) {
  /*represents grey-70 */
  color:  #4a4b57 !important;
}

:root legend span:not([class*="error"]) {
  color:  #4a4b57 !important;
}
```

I thought you added that change to make it work for the messages view? 🤔 

WHATEVER :D it looks good, it's clearly scoped so feel free to hit MERGE.